aten/src/ATen/native/sparse/SparseCsrTensorMath.cpp

```diff
@@ -94,6 +94,9 @@
 #include <ATen/ops/zeros.h>
 #endif
 
+#include <ATen/ops/_unique.h>
+#include <ATen/ops/tensor.h>
```

Comment:
FYI: These need to come before the `#endif` above. That's why the `no-ops` CI is failing.

Reply:
Fixed. Thanks, @peterbell10 !